dataclients/kubernetes/routegroup.go

```diff
@@ -0,0 +1,554 @@
+package kubernetes
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/loadbalancer"
+)
+
+// TODO:
+// - consider catchall for east-west routes
+// - document how route group errors are handled
+// - document in the CRD that the service type must be ClusterIP when using service backends
+// - document the implicit routes, or clarify: spec.routes is not optional, but an example doesn't have any
+// - document the rules and the loopholes with the host catch-all routes
+// - document the behavior of the weight implementation
```

Comment:
do you already documented all of these?
If not do you plan to do this in this PR?

Reply:
yes, removing these todo items.